Libraries/StyleSheet/ColorValueTypes.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import type {NativeColorValue} from './PlatformColorValueTypes';
```

Comment:
I think this can be merged into the statement below:

```
import {
  PlatformColor,
  normalizeColorObject,
  processColorObject,
  type NativeColorValue,
} from './PlatformColorValueTypes';
```

Reply:
Got it.